CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 6.0.1
```

Comment:
Yep, I give you wrong advice ... this PR should be merged before tagging. 
Since I still stand on my opinion to not remove already created tag, we might use v6.0.2